### PR TITLE
portfolio setup Mobile first

### PR DESCRIPTION
**Pull request by ariful04**
In this branch I did:
- [ ] Add HTML file.
- [ ] Add CSS file.
- [ ] Edit Readme file.
- [ ] Add Linter
I try to design as possible as Figma template.

### CODE REVIEW

**Review by topeogunleye — COMMENTED** [review]
Hi @ariful04, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that the README file is not complete. Please use the [Microverse template](https://github.com/microverseinc/readme-template).

- [ ] In the case of README consider not leaving out any of the sections in the [Microverse README](https://github.com/microverseinc/readme-template) template as it's a project requirement.

| professional Readme  | Your Readme that is not professional |
| ------------- | ------------- |
|  Recording 2023-02-14 at 12 52 20 (5)	  | image |

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by ariful04** [comment]
Hello @topeogunleye  I update my Readme file, Please recheck my pull request.

**Comment by topeogunleye** [comment]
> Hello @topeogunleye I update my Readme file, Please recheck my pull request.

You deleted the README file

**Comment by ariful04** [comment]
Hello @topeogunleye I solved the problem in Readme file, Please recheck my pull request.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    <main class="main-content">
+        <section class="headline">
+            <h1 class="headline-title">I am Md Ariful Glad to see you!</h1>
+            <p class="headline-content">I'm a software developer I can help you
+                build a product feature or website Look through some of my works and
+                experience! If you like what you see and have a project you need coded
+                don't hesitate to contact me.
+            </p>
+         <div class="headline-social">
+            <h3 class="social-title">LET'S CONNECT</h3>
```
Comment (TEMESGEN-G25) [inline index.html:33]: - [ ] Great job! :rocket: On matching the Figma design, but there's just one slight problem. Currently, you're not using the correct font for your content. I recommend using _Poppins_. That way you'll have a much prettier looking UI.

**Review by TEMESGEN-G25 — CHANGES_REQUESTED** [review]
## :yellow_circle: CHANGES REQUIRED :yellow_circle: 

## Hi @ariful04 :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Used the correct elements for the correct content.
:heavy_check_mark: Used background images instead of an `img` tag



## Required Changes ♻️
_Check the comments under the review._
:x: On your README file the authors' section doesn't seem to have proper links set up. I recommend that you fix it so that other senior devs or HRs would be able to reach you. I also suggest fixing the LICENSE link because, currently, it's displaying an error page.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please consider:
:x: One of the general requirements is to follow GitHub flow. That means that the master branch should have no commits except for one commit, which is for creating the README file. All other commits or changes should be part of the feature branch and then merged into the master branch once it has been approved. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ariful04) [reply index.html:33]: Hello reviewer I update my Readme file as requirement in MIT licence link, Auther section proper link, used Poppins font in css file please recheck my pull request.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        </nav>
+    </header>
+    <main class="main-content">
+        <section class="headline">
+            <h1 class="headline-title">I am Md Ariful Glad to see you!</h1>
+            <p class="headline-content">I'm a software developer I can help you
+                build a product feature or website Look through some of my works and
+                experience! If you like what you see and have a project you need coded
+                don't hesitate to contact me.
+            </p>
```
Comment (OyePriscilla) [inline index.html:31]: - [ ] So sorry, I guess this is an oversight! You suppose to use the correct font-family (`poppins`) for this paragraph as you rightly did for the `headline title` as pointed out earlier. 👍

File: README.md [hunk 3]
```diff
@@ -160,7 +160,7 @@ For install Linter go to this link Here describe all step.
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](https://choosealicense.com/licenses/mit/) licensed.
 
 _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (OyePriscilla) [inline README.md]: - [optional] Kindly remove this template text, it is there to direct on what to do with this section. ℹ️

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @ariful04 👏 👏, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so-close-so-close-seth-meyers

 ## Highlights
Added License file to README 👍 
Author section of the README file redirects appropraitely ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (ariful04) [reply README.md]: Hello @OyePriscilla Thanks for fast review, I change Readme file and css file as your requirement please check.

File: index.html
Comment (Kamba56) [inline index.html:1]: - [Optional] COnsider using an indentation of two spaces in all your documents. THis is a convention as well as best practice.

**Review by Kamba56 — APPROVED** [review]
Hi @ariful04, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
## Highlights
- [x] Great job adding the header and headline sections!👏👏👏
- [x] Great job using CSS flexbox!👏👏👏
- [x] Descriptive PR!👏👏👏

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**